Example/SwiftUIExample/MobilePaymentsSwiftUIExample/Screens/Home/HomeView.swift

```diff
@@ -0,0 +1,220 @@
+import SquareMobilePaymentsSDK
+import SwiftUI
+
+#if canImport(MockReaderUI)
+import MockReaderUI
+#endif
+
+struct HomeView: View {
+
+    @State private var presentingPermissionsView: Bool = false
+    @State var viewModel: HomeViewModel
```

Comment:
Can this be made private? Same with the PermissionsView viewModel?

Reply:
Updated 🙏